packages/core/content-type-builder/admin/src/components/FormModal/attributes/validation/common.js

```diff
@@ -14,7 +14,17 @@ const alreadyUsedAttributeNames = (usedNames) => {
         return false;
       }
 
-      return !usedNames.includes(value);
+      let alreadyUsed = false;
+      for (let i = 0; i < usedNames.length; i++) {
+        const name = usedNames[i];
+
+        if (toLower(name) === toLower(value)) {
+          alreadyUsed = true;
+          break;
+        }
+      }
+
+      return alreadyUsed;
```

Comment:
very well, it is a little of overhead but I think is acceptable.

Minor thing, we could make it a little bit more easy to read:
```suggestion
      return !usedNames.map(toLower).includes(value);
```

Reply:
I was worrying about the performance thus decided not to use es6 `map` with `includes`, thought `for loop` would be faster :D


but you're right, `map` with includes is waay  faster, and it's easier to read as well. I will commit the changes.
<img width="1166" alt="Screen Shot 2022-10-20 at 10 03 13 AM" src="https://user-images.githubusercontent.com/46695441/196891782-12f070a6-e354-4fcb-97c8-52a09c02a45f.png">

Reply:
oh, never done a performance test like that! Very interesting 👀

Reply:
it helps a lot, [benshmark ](https://www.npmjs.com/package/benchmark) is the magic behind this